tails_files/install.sh

```diff
@@ -21,6 +21,13 @@ DESKTOP=$HOMEDIR/Desktop
 ANSIBLE=$PERSISTENT/securedrop/install_files/ansible-base
 SSH_ALIASES=false
 
+# check for Tails 2.x
+source /etc/os-release
```

Comment:
In this case, sourcing `/etc/os-release` seems fine to me. It appears to have been designed with that purpose in mind: https://www.freedesktop.org/software/systemd/man/os-release.html says

> It is possible to source the configuration from shell scripts